google_shopping.js

```diff
@@ -1,5 +1,68 @@
 var data = require("./products.json")
 
-// Write your solutions below
+// Solution for No. 1
+
+var count = 0
+
+for (var i = 0; i < data.items.length; i++) {
+	if (data.items[i].kind === "shopping#product") {
+		count ++;
+		// console.log("Yes")
+	}
+	// console.log(data.items[i])
+}
+
+console.log(count);
+
+
+// Solution for No. 2
+
+for (var i = 0; i <data.items.length; i++) {
+	if (data.items[i].product.inventories[0].availability === "backorder") {
+		console.log(data.items[i].product.author.name);
```

Comment:
Instead of console logging the product author name, perhaps logging the item name would be a better indicator.